examples/contracts/aggregator-consumer/scripts/getLatestPrice.ts

```diff
@@ -0,0 +1,25 @@
+import { starknet } from 'hardhat'
```

Comment:
why is this file named `getLatestPrice`?

Reply:
@qperrot

Reply:
Same thing. But this is currently not a good implementation, I had to finish the ERC677 first, then I will go back on those exemple.

Reply:
@cds95